js/pages/homePage.js

```diff
@@ -167,107 +167,479 @@ export const whereAmIInDashboard = async () => {
+                         <p>Researchers interested in studying information collected through the Connect for Cancer Prevention Study can <a href="https://dceg.cancer.gov/research/who-we-study/cohorts/connect">learn more about the cohort</a> and how to request access to shared resources.</p>
+                      </section>
+                   </div>
+                </div>
+             </div>
+             <footer class="article-footer">
+                <div id="nvcgSlSyndication">
+                   <div class="contentid-911760 slot-item only-SI">
+                      <div class="syndication">
+                      </div>
+                   </div>
+                </div>
+             </footer>
+          </article>
+       </div>
+       <div class="col-1">
+       </div>
+    </div>     `
     window.scrollTo(0, 0);
 }
 
 export const renderHomeExpectationsPage = () => {
     const mainContent = document.getElementById('root');
     mainContent.innerHTML = `
-        <div class="row">
-        <div class="col-1">
-        </div>
-        <div class="col-10" id="connectBody">
-        </div>
-        <div class="col-1">
-        </div>
+    <link rel="stylesheet" media="all" href="https://www.cancer.gov/connect-prevention-study/sites/g/files/xnrzdm246/files/css/css_OLbt7L-WJSHlUUD5jXU697lzYlbsp4N6NkV-x07NBpI.css"/>
+    <link rel="stylesheet" media="all" href="https://www.cancer.gov/connect-prevention-study/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/dist/css/Common.css?ryhhto"/>
+    <link rel="stylesheet" media="all" href="https://www.cancer.gov/connect-prevention-study/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/dist/css/Article.css?ryhhto"/>
+    <link rel="stylesheet" media="all" href="https://www.cancer.gov/connect-prevention-study/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/dist/css/Connect.css?ryhhto"/>
+    <div class="row">
+       <div class="col-1">
+       </div>
+       <div class="col-10" id="connectBody">
+          <article>
+        <div class="resize-content">
+          <h1>
+              What to Expect
+          </h1>
+          <div class="page-options-container">
+              <!-- PAGE OPTIONS -->
+              <div id="PageOptionsControl1" class="page-options no-resize">
+                <ul>
+                    <li class="page-options--print">
+                      <a title="Print" aria-label="Print" href="#"></a>
+                    </li>
+                    <li class="page-options--email">
+                      <a title="Email" aria-label="Email" href="#"></a>
+                    </li>
+                </ul>
+              </div>
+              <!-- END PAGE OPTIONS -->
+          </div>
```

Comment:
remove this div. These are not in use and they're rendering as bullet points with no text. In the current format, this disappears when the stylesheet is fetched and loaded, but it is visible momentarily beforehand.